Sources/TokamakStaticHTML/Resources/TokamakStyles.swift

```diff
@@ -76,6 +76,34 @@ public let tokamakStyles = """
   height: 1.2em;
   border-radius: .1em;
 }
+
+@media (prefers-color-scheme:dark) {
+  ._tokamak-buttonstyle-default {
+    background-color: rgb(99, 95, 98);
```

Comment:
Does this work with native-looking buttons in Safari?

Reply:
<img width="104" alt="Screen Shot 2020-08-02 at 10 07 42 PM" src="https://user-images.githubusercontent.com/13581484/89139429-97b88e80-d50c-11ea-92c0-d279977ae434.png">

Overwrites the style, just like Chrome. Is there a better way to do this besides trying to replicate the control?

Reply:
Yeah, you can set `color-scheme: light dark` on the `<html>` element or on the button specifically to get a native dark button in dark mode.